plugins/htpasswd/CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing
```

Comment:
With CONTRIBUTING we can do 2 things: remove and improve outer CONTRIBUTING or refactor this one to be specific to the project. I prefer the first one, but what do you think @verdaccio/core-team and @griffithtp ?

Reply:
Yes, one CONTRIBUTING is enough.

Reply:
one CONTRIBUTING, unless different packages has different specific contributing notes